cmd/session-migrate.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 // Migrates session header version '7' to '8'. The only
-// change was the adding of insecure global flag
+// change was the adding of skipCertCheck global flag
 func migrateSessionV7ToV8() {
```

Comment:
I think you need to create a new session version V9 and migrate from V8 to V9 by renaming insecure to skipCertCheck

Reply:
yes you have to @Praveenrajmani

Reply:
Done